### ISSUE TITLE
[OBS Studio 27 RC1] Source filters are not applied during transition

### ISSUE THREAD

**Issue by Brawaru** [exeldro/obs-move-transition#71]
When switching scenes, source filters are no longer applied during the transition, resulting in blinking if you have colour correction, for example. It broke in 27-RC1, so it's probably worth reporting it now to prepare for the 27 release.

**Comment by exeldro** [comment]
For me the color correction on a source seems to be working as normal using a move transition, have you maybe added the color correction to the scene (#2)?

**Comment by Brawaru** [comment]
Not really, no. What's weirder, I'm unable to reproduce it with a new, empty scene collection.

Actually, yes. I've just tested: re-adding elements does not reproduce the issue. I guess it's a side effect of updating to 27 RC1. Giving, this update already messed up with a colour correction.

Here's a demonstration of what's happening: I've re-added backgrounds for Among Us and Minecraft, but haven't changed Portal one. Filters are the same as before the upgrade. Notice how the old Portal background blinks after the transition ends.

https://user-images.githubusercontent.com/10401817/113505551-cecdbc80-953f-11eb-8a6e-7a375bc40442.mp4

I can only guess now it's actually OBS' issue…
